src/h5web/visualizations/shared/AxisGrid.tsx

```diff
@@ -63,12 +67,12 @@ function AxisGrid(): JSX.Element {
       }}
     >
       <>
-        <IndexAxis
+        <CustomTickAxis
```

Comment:
You could just call it `Axis` to be honest. You'd just have to rename the variable inside the component where we store VX's `LeftAxis` or `RightAxis`, perhaps to `DirectionalAxis`?